src/services/validation.js

```diff
@@ -22,9 +24,11 @@ export async function validateSRN(userID, validateCount, isSingleEntryOnly, redi
     //this condition checks the second time, since still not valid, just changes the flag and continues with the plio.
     else if (!doesUserExist && validateCount == 1) {
     validateCount = 2;
+    sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType);
     sendPlio(isSingleEntryOnly, userID, redirectID);
 
     } else {
+    sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType);
```

Comment:
Am I missing something? The `elseif` and `else` seem to have the exact same code except for `validateCount`... I feel like this can be simplified. 

Very complex logic :) But maybe no need to simplify. Is this deployed on staging already, and can I test?